packages/hashlib/meta.yaml

```diff
@@ -5,11 +5,12 @@ package:
     - always
   top-level:
     - _hashlib
+source:
+  sha256: 065040198fde6fd7229b8c0538fc8b848824e35930e3904f46c9e90e9f0f1a3f
+  url: https://github.com/python/cpython/archive/refs/tags/v$(PYVERSION).tar.gz
```

Comment:
Are you sure it's safe to take these from the original archive and compile them without running the configure step? Isn't there a risk something would be badly configured then?

Reply:
That's a good point. I checked that the files are not changed before/after configure (with [this simple script](https://gist.github.com/ryanking13/2b8ab6af2b9c8423f58b0394e311b092)). And we are running full CPython test suites for unvendored packages for hopefully it is okay...